src/views/Invite/InviteEmail.js

```diff
@@ -189,24 +189,30 @@ export default function InviteEmail() {
     }
 
     //TODO want to do this on-input, but that gets weird. see #188
-    let errorCount = 0;
+    let knowAll = true;
+    let alreadyReceived = [];
     await Promise.all(
       inputs.map(async input => {
         const email = input.data;
         const hasReceived = getHasReceived(email).matchWith({
-          Nothing: () => false, // loading
+          Nothing: () => 'unknown', // loading
```

Comment:
I would just do something like

```
const knowAll = every(inputs.map(input => getHasReceived(input.data).getOrElse(false))
const alreadyReceived = inputs.map(input => getHasReceived(input.data).getOrElse(false) ? input.data : null).filter(x => !!x);

...
```

Reply:
We discussed (out of band) doing this with `Result` instead, but both that and the current solution just end up taking needless extra steps. Pushed a version that gets to the point more directly.